### PR TITLE
STORM-1242: migrate backtype.storm.command.config-value to java

### PR DESCRIPTION
**Pull request by revans2**

### CODE REVIEW

File: storm-core/src/jvm/org/apache/storm/command/ConfigValue.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ */
+package org.apache.storm.command;
+
+import java.util.Map;
+
+import org.apache.storm.utils.ConfigUtils;
+
+public class ConfigValue {
+    public static void main(String [] args) {
+        String name = args[0];
```
Comment (redsanket) [inline storm-core/src/jvm/org/apache/storm/command/ConfigValue.java]: do we need to check for presence of elements before accessing them.

**Comment by redsanket** [comment]
LGTM, minor clarification

File: storm-core/src/jvm/org/apache/storm/command/ConfigValue.java (hunk 1 above)
Comment (revans2) [inline storm-core/src/jvm/org/apache/storm/command/ConfigValue.java]: The old code didn't, and the python code blows up even worse if you don't pass in the parameter.

**Comment by kishorvpatil** [comment]
+1

**Pull request merged**